src/api/render-function.md

```diff
@@ -228,33 +228,33 @@ For manually resolving a registered component by name.
 
   </div>
 
-- **See also** [Guide - Render Functions - Components](/guide/extras/render-function#components)
+- **Guarda anche** [Guide - Render Functions - Components](/guide/extras/render-function#components)
 
 ## resolveDirective() {#resolvedirective}
 
 For manually resolving a registered directive by name.
```

Comment:
For manually resolving a registered directive by name. 

-> 
Per risolvere manualmente una direttiva registrata per nome.